### PR TITLE
[Easy] npm audit fix

### PR DESCRIPTION
**Pull request by bh2smith**
Repairing all vulnerabilities that can be fixed by package upgrade.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "eslint": "^5.12.0",
     "eslint-plugin-no-only-tests": "^2.3.0",
     "eslint-plugin-react": "^7.12.3",
-    "eth-gas-reporter": "^0.2.0",
+    "eth-gas-reporter": "^0.2.1",
```
Comment (bh2smith) [inline package.json]:
```suggestion
    "eth-gas-reporter": "^0.2.0",
```

**Comment by fleupold** [comment]
What happened, why did you close the issue?

**Comment by bh2smith** [comment]
I want to make sure it is working first. Also I thought it was the cause of the 🐛 I reported (but no longer believe this to be true).

**Comment by bh2smith** [comment]
This build is failing because of 

```
npm ERR! code ENOLOCAL
npm ERR! Could not install from "node_modules/eth-gas-reporter/node_modules/abi-decoder/node_modules/web3/bignumber.js@git+https:/github.com/debris/bignumber.js.git#94d7146671b9719e00a09c29b01a691bc85048c2" as it does not contain a package.json file.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-06-24T18_32_33_404Z-debug.log
```
which appears to be related to [this](https://github.com/npm/npm/issues/18266) issue. Will look into bignumber and see what the deal is.

**Comment by bh2smith** [comment]
I have no idea what to do here. Do either of you @josojo or @fleupold ?

**Comment by fleupold** [comment]
> I have no idea what to do here. Do either of you @josojo or @fleupold ?

You could create a PR in the `eth-gas-reporter` repo trying to change the version of abi-decoder to something that works?

But you are right that the error doesn't make sense. https:/github.com/debris/bignumber.js.git#94d7146671b9719e00a09c29b01a691bc85048c2 does include a `package.json`

**Review by josojo — COMMENTED** [review]
I cannot assess this one.

**Comment by cgewecke** [comment]
Out of curiosity is Denis Granha (the maintainer of abi-decoder) still at Gnosis? There are PRs open in [that repo](https://github.com/ConsenSys/abi-decoder/pulls) to upgrade the web3 version which could resolve this problem at it's source. I'd rather not find another decoding solution - it's a really nice package. 

Also, web3 1.0-beta.37 has recently been selected as the [official 'stable'](https://github.com/ethereum/web3.js/pull/2905) version of web3 1.0 and a PR for that release is in progress. This might be an opportune time to revisit abi-decoder maintenance.

**Comment by bh2smith** [comment]
Thanks for the suggestions @cgewecke, I'll get in touch with @denisgranha about this one!

**Comment by bh2smith** [comment]
So finally, the web3 dependency in abi-decoder was updated and this fix no longer depends on using yarn.

I am unsure what is the deal with displaying solidity-bytes utils update. This has clearly already been changes in master so should not appear as different here.

**Review by fleupold — APPROVED** [review]
Would love to see the unrelated travis changes be in a separate PR but otherwise this seems good to go.

⬆️

**Pull request merged**